lock-impl/src/main/java/com/palantir/lock/logger/LogState.java

```diff
@@ -37,13 +36,13 @@ public interface LogState {
     Multimap<ObfuscatedLockDescriptor, SimpleTokenInfo> getHeldLocks();
 
     @Safe
-    Map<ObfuscatedLockDescriptor, String> getSyncState();
+    Multimap<ObfuscatedLockDescriptor, String> getSyncState();
 
     @Safe
-    Map<ClientId, List<SanitizedLockRequestProgress>> getSynthesizedRequestState();
+    Multimap<ClientId, SanitizedLockRequestProgress> getSynthesizedRequestState();
 
     @Unsafe
-    Map<ObfuscatedLockDescriptor, LockDescriptor> getLockDescriptorMapping();
+    Multimap<ObfuscatedLockDescriptor, LockDescriptor> getLockDescriptorMapping();
```

Comment:
I think this one is supposed to be 1:1. This is the mapping from safeloggable "Lock-3" to the unsafe lock descriptor string, which doesn't include r/w in it?

Reply:
will revert